apps/home/templates/svg/explanation.html

```diff
@@ -0,0 +1,5 @@
+<svg class="svg_icon" width="114" height="100" viewBox="0 0 114 100" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Can we keep the file names matching the content? If we use `explanation.svg` desktop apps can open the file.

FYI, Django templating doesn't care about the filename.

Reply:
Did we optimise the SVGs? https://jakearchibald.github.io/svgomg/

Reply:
Ohh nice. I didn't know that we could use `{% include svg/explanation.svg %}`. Fine I will rename `templates/svg/*.html` to `templates/svg/*.svg`. 
Also I thought that we weren't sure of optimisations yet and therefore I didn't optimise. I will optimise and include in next commit.